api/src/pcapi/core/offerers/repository.py

```diff
@@ -562,14 +563,28 @@ def get_offerer_and_extradata(offerer_id: int) -> models.Offerer | None:
         .correlate(models.Offerer)
         .exists()
     )
-
+    has_account_with_pending_corrections_subquery = (
+        sqla.select(1)
+        .select_from(finance_models.BankAccount)
+        .where(
+            sqla.and_(
+                finance_models.BankAccount.offererId == models.Offerer.id,
+                finance_models.BankAccount.isActive.is_(True),
+                finance_models.BankAccount.status
+                == finance_models.BankAccountApplicationStatus.WITH_PENDING_CORRECTIONS,
+            ),
+        )
+        .correlate(models.Offerer)
+        .exists()
+    )
     return (
         db.session.query(
             models.Offerer,
             sqla.or_(has_non_free_offers_subquery, has_non_free_collective_offers_subquery).label("hasNonFreeOffer"),
             has_valid_bank_account_subquery.label("hasValidBankAccount"),
             has_pending_bank_account_subquery.label("hasPendingBankAccount"),
             has_active_offers_subquery.label("hasActiveOffer"),
+            has_account_with_pending_corrections_subquery.label("hasAccountWithPendingCorrections"),
```

Comment:
[détail / nommage] Puisque les autres labels mentionnent "**bank** account" dans leur nom, et pour éviter une éventuelle confusion avec des "_user_ accounts", je suggère : 

```suggestion
            has_account_with_pending_corrections_subquery.label("hasBankAccountWithPendingCorrections"),
```

Reply:
Je relais la proposotion au lead tech( vu que j'avais déjà laissé un commentaire sur le ticket). Mais pour moi ça rentre effectivement bien dans une logique de non confusion.